### PR TITLE
[Doc Improvement][Add a limitation about embedding an existing bot n Teams][2760360]

### PR DESCRIPTION
**Pull request by v-npaladugu**

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/what-are-bots.md](https://github.com/MicrosoftDocs/msteams-docs/blob/11a1908ba04951c574d755d7bf531737ac76133e/msteams-platform%2Fbots%2Fwhat-are-bots.md) | 93 | 5 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/85f240dc-108a-4c81-8a86-1dd97cede073) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/what-are-bots.md](https://github.com/MicrosoftDocs/msteams-docs/blob/82a450549475e8f0a7874aad66cfaa85df9a85e7/msteams-platform%2Fbots%2Fwhat-are-bots.md) | 93 | 5 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/5e752c5d-8047-460b-8495-a2e26a56ddd3) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/what-are-bots.md](https://github.com/MicrosoftDocs/msteams-docs/blob/263f7fb62dcdc7b1a56897b8d9dada4e0a3843e2/msteams-platform%2Fbots%2Fwhat-are-bots.md) | 94 | 4 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/96542273-3085-4c24-a7ec-90880a2b7fb2) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/bots/what-are-bots.md [hunk 1]
```diff
@@ -49,6 +49,8 @@ With Microsoft Graph APIs for calls and online meetings, Teams apps can now inte
 
 You can use the Teams bot APIs to get information for members of a chat or team. See [changes to Teams bot APIs for fetching team or chat members](~/resources/team-chat-member-api-changes.md).
 
+Embedding an existing bot as a Teams bot isn't possible. You can create a new Teams bot by using the back-end infrastructure and your business logic.
```
Comment (ashishguptaiitb) [inline msteams-platform/bots/what-are-bots.md:52]: @v-npaladugu , Wajeed and I discussed more than what's reflected in this line. I recommend taking notes please.

1. Mention first that for security reasons, apps do not allow as is embedding of any existing bot. 
1. This means that dev have to create a new bot for their Teams app.
1. However, to make their life easier, they can reuse the business logic + backend infrastructure in the Teams bot.

Comment (ashishguptaiitb) [reply msteams-platform/bots/what-are-bots.md:52]: An update @v-npaladugu , do not mention anything about this particular issue. Also, remove the edit `Embedding an existing bot as a Teams bot isn't possible. You can create a new Teams bot by using the back-end infrastructure and your business logic.` I've updated our shared spreadsheet as well.

**Comment by surbhigupta12** [comment]
Closing this PR as the content need not be published.

**Pull request closed without merging**